src/theme/index.ts

```diff
@@ -1,4 +1,5 @@
 import { createTheme } from "@mui/material";
+import type {} from "@mui/x-data-grid/themeAugmentation";
```

Comment:
Is this needed?

Reply:
It is, as this provides MuiDataGrid theme types via type augmentation to ThemeOptions, which is now required in createTheme where we provide the default theming to the DataGrid.